client/src/app/tabs/dmn/DmnEditor.js

```diff
@@ -391,7 +391,8 @@ export class DmnEditor extends CachedComponent {
         defaultCopyCutPaste: true,
         defaultUndoRedo: true,
         removeSelected: true,
-        selectAll: true
+        selectAll: true,
+        inputActive: true
```

Comment:
I believe if we go for such a hack we should properly document it in the source code, i.e. by providing a link back  to the PR.

It would otherwise cost us a considerable amount some time to understand why `inputActive` has to be there in the first place.

Reply:
Done :heavy_check_mark: